test/graphtile.cc

```diff
@@ -32,6 +32,12 @@ TEST(Graphtile, FileSuffix) {
   EXPECT_THROW(GraphTile::FileSuffix(GraphId(1036800, 2, 0)), std::runtime_error);
   EXPECT_THROW(GraphTile::FileSuffix(GraphId(4050, 0, 0)), std::runtime_error);
   EXPECT_THROW(GraphTile::FileSuffix(GraphId(1036800, 3, 0)), std::runtime_error);
+
+  TileLevel level{7, valhalla::baldr::RoadClass::kSecondary, "half_degree_is_a_multiple_of_3",
+                  Tiles<PointLL>{{{-180, -90}, {180, 90}}, .5, 1}};
+  EXPECT_EQ(GraphTile::FileSuffix(GraphId(1234, 7, 0), ".qux", false, &level), "7/001/234.qux");
+  ASSERT_EXIT((GraphTile::FileSuffix(GraphId(123456, 7, 0), ".qux", false, &level),exit(0)),::testing::ExitedWithCode(0),".*");
```

Comment:
You can remove this one I guess. The point was that the below ID should fail master, we can't verify that anymore on this branch since you fixed it. It's enough for us when you say "jep, the below test fails master". Just verified it myself, you can safely remove this test line.